scipy/linalg/basic.py

```diff
@@ -41,8 +41,8 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
           overwrite_b=False, debug=None, check_finite=True, assume_a='gen',
           transposed=False):
     """
-    Solves the linear equation set ``a * x = b`` for the unknown ``x``
-    for square ``a`` matrix.
+    Solves the linear equation set ``ax = b`` for the unknown ``x`` for
```

Comment:
```suggestion
    Solves the linear equation set ``a@x = b`` for the unknown ``x`` for
```
I agree that `*` was not the right operator, but if this is going to be written as code (between double backticks), consider using the closest operator. I understand that this isn't perfect (we're not using `=` as an assignment operator here) but it's closer.

Reply:
How about changing it to single backticks and write `a x = b` or `ax = b` (as in [numpy.linalg.solve](https://numpy.org/doc/stable/reference/generated/numpy.linalg.solve.html))?
Personally, I'd prefer to use a more mathematical syntax in the documentation over Python syntax.
But I'm also fine with ``a @ x = b``, if this is preferred way of expressing mathematical equations in the scipy docs.